Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -159,6 +154,19 @@ function Invoke-ConfigTransform {
 
         Invoke-TransformConfigFile -sourceFile $baseConfig -transformFiles $transformFiles  -destinationFile $destinationFile
     }
+
+    Invoke-Task -name $MyInvocation.MyCommand.Name -task {
+        $baseConfig = Get-RepositoryResolvedPath 'Application\EdFi.Ods.SandboxAdmin.Web\Web.Base.config'
+        $npgsqlTransform = Get-RepositoryResolvedPath 'Application\EdFi.Ods.SandboxAdmin.Web\Web.Npgsql.config'
+        $debugTransform = Get-RepositoryResolvedPath 'Application\EdFi.Ods.SandboxAdmin.Web\Web.Debug.config'
+        $destinationFile = ($baseConfig -replace "base.", "")
+
+        $transformFiles = @()
+        if ($Engine -eq "PostgreSQL") { $transformFiles += $npgsqlTransform }
+        $transformFiles += $debugTransform
+
+        Invoke-TransformConfigFile -sourceFile $baseConfig -transformFiles $transformFiles  -destinationFile $destinationFile
+    }
```

Comment:
This looks like a duplicate of the task above.

Reply:
this does the sandbox config transform